README.md

```diff
@@ -41,14 +41,37 @@ For understanding the overall objective of the library please refer to [openmine
 
 ### Demo App
 
-To run the demo app, create a file `local_config.properties` under the folder `demo-app` and add the following line:
+Follow these steps to setup an environment where to test the demo app.
+
+
+1. Clone the repo https://github.com/OpenMined/PyGrid
+2. `cd PyGrid`
+3. Install docker: https://github.com/OpenMined/PyGrid/#getting-started
+4. Install docker-compose.
+5. Execute `docker-compose up` in the command line.
+6. Clone the repo https://github.com/OpenMined/PySyft
```

Comment:
For pysyft we are currently not compatible with latest plan. We need to use a previous version. Temporarily we can refer to https://github.com/vkkhare/PySyft/tree/Kotlinsyft

Reply:
What I have done in the past is to add a section indicating which version of PySyft is compatible with the current development.
That can be a specific commit in PySyft or a release tag.

Reply:
There is a specific commit but there are also changes in the notebook file. We can push this to pysyft repo but I am anyway in the process of refactoring so this would not stay for very long